### PR TITLE
mobile friendly mods

### PR DESCRIPTION
**Pull request by LateBy5**
Updated the memory component and cpu page to be slightly more mobile friendly. Tested various viewports out locally - changes look alright.

### CODE REVIEW

File: src/pages/cpu.ts [hunk 1]
```diff
@@ -69,23 +69,27 @@ export const CPU = (
...
-            justifyContent: "center",
+            gridTemplateColumns: "repeat(4, minmax(125px, 283px))",
+            justifyContent: "center"
           },
           "hack-screen": {
-            gridColumn: "3 / span 2",
-          },
+            width: "100%",
+            maxWidth: "600px",
+            gridColumn: "3 / span 4",
```
Comment (DavidSouther) [inline src/pages/cpu.ts:78]: It looks like this can be 3 / span 2

File: src/pages/cpu.ts [hunk 2]
```diff
@@ -69,23 +69,27 @@ export const CPU = (
...
+          }
         },
         "@media (max-width: 1195px)": {
```
Comment (DavidSouther) [inline src/pages/cpu.ts:81]: I know you found it this way, but this is backwards - we want to only use the min-width media selectors, inline with the breakpoints in picocss.

File: src/pages/cpu.ts [hunk 3]
```diff
@@ -69,23 +69,27 @@ export const CPU = (
...
           "div.grid": {
-            gridTemplateColumns: "repeat(4, 283px)",
-            justifyContent: "center",
+            gridTemplateColumns: "repeat(4, minmax(125px, 283px))",
+            justifyContent: "center"
           },
           "hack-screen": {
-            gridColumn: "3 / span 2",
-          },
+            width: "100%",
```
Comment (DavidSouther) [inline src/pages/cpu.ts:76]: This width turns out we don't need

File: src/components/screen.ts [hunk 4]
```diff
@@ -56,11 +55,12 @@ export const Screen = FC<
     figure(
       {
         style: {
+          width: "100%",
+          marginInline: "auto",
```
Comment (DavidSouther) [inline src/components/screen.ts:59]: marginInline is the secret sauce, here, and we can remove the width to keep the border tight around the canvas

**Review by DavidSouther — CHANGES_REQUESTED** [review]

File: src/pages/cpu.ts (hunk 2 above)
Comment (LateBy5) [reply src/pages/cpu.ts]: Our version of pico css doesn't seem to contain media queries for the container class, so I shimmed them in the js for the time being

File: src/components/screen.ts (hunk 4 above)
Comment (LateBy5) [reply src/components/screen.ts]: > marginInline is the secret sauce, here, and we can remove the width to keep the border tight around the canvas

Turns out we don't need either. Instead I'm using text-align to center the canvas within the figure element - that way nothing gets screwy with the horizontal scrolling

**Comment by LateBy5** [comment]
Per your initial comments, I took a first look at pico css (lol - should of did that from the start). I updated the styling to use / align with the ethos of pico css

File: src/pages/cpu.ts (hunk 2 above)
Comment (DavidSouther) [reply src/pages/cpu.ts]: We could add containers using the query params to pico, but this is I think better without them.

File: src/components/screen.ts (hunk 4 above)
Comment (DavidSouther) [reply src/components/screen.ts]: Great - and then, the last piece is the border stays visible but doesn't get wider than the canvas.

**Review by DavidSouther — APPROVED** [review]

**Pull request merged**